README.md

```
@@ -1,74 +1,63 @@
 
-   ```bash
-   bun install
-   ```
+## Project Layout
 
-3. **Run the development server:**
-   ```bash
-   bun run dev
-   ```
-   The application will be available at `http://localhost:5173`.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)